tracer/test/Datadog.Trace.Security.IntegrationTests/RASP/AspNetCore2Rasp.cs

```diff
@@ -92,5 +92,24 @@ public async Task TestRaspRequest(string url, string exploit)
         settings.AddIastScrubbing();
         await VerifySpans(spansFiltered.ToImmutableList(), settings, testName: testName, methodNameOverride: exploit);
     }
+
+    [SkippableTheory]
+    [Trait("Category", "ArmUnsupported")]
```

Comment:
Why don't we support ARM?

Reply:
Because we are using Sqlite which is not fully compatible. We also do the same in other IAST / tracer tests